miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -51,7 +51,7 @@ internal open class MiniAppWebView(
     }
 
     init {
-        if (this::class.java.simpleName == "MiniAppWebView")
+        if (this::class.java == MiniAppWebView::class.java)
```

Comment:
Shouldn't this also work with `this::class == MiniAppWebView::class`? But I'm not sure if there's any advantage of using `::class` or `::class.java` 🤔  

If it's working with `::class.java` then probably it's okay.

Reply:
I have updated to `::class`.

Reply:
```
Output of `this::class` or `this::class.java`:

without obfuscation: `com.rakuten.tech.mobile.miniapp.display.MiniAppWebView`
with obfuscation: `com.rakuten.tech.mobile.miniapp.display.d`
```
Previously, it causes the issue because, output of `this::class.java.simpleName` was "d" with obfuscation.

However, let's merge this as a patch.  My suggestion is doing something else while comparing these values, so maybe we can improve this later.